prepare-videochat.sh

```diff
@@ -241,7 +245,11 @@ url_reachable() {
         # has it in its core, so we don't need to check that case)
         sudo apt-get install -y curl
     fi
-    curl -m 5 -sI "$1" >/dev/null
+    if [ $DISABLE_PROXY = 1 ]; then
+        curl --noproxy '*' -m 5 -sI "$1" >/dev/null
+    else
+        curl -m 5 -sI "$1" >/dev/null
+    fi
```

Comment:
It's better to use if..then to set a CURL_OPTIONS variable to either "--noproxy" or "", and then use `curl $CURL_OPTIONS ...`, to avoid duplicating the same command except for that flag.